From ebd4eab3b859968268483887d2bf96d1c5a28759 Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Wed, 14 Nov 2018 22:33:55 -0800
Subject: [PATCH] Fix warning printed when /lib or /lib64 are symlinks

icecc-create-env was printing:
[...]
adding file /usr/bin/objcopy
adding file /etc/ld.so.conf=/tmp/icecc_ld_so_confTsSNqs
cp: -r not specified; omitting directory '/lib'
cp: -r not specified; omitting directory '/lib64'
creating d7801039695fbb794e026bdaf287276c.tar.gz

The -p option means to preserve ownership. To copy the symlink, we want
-d.
---
 client/icecc-create-env.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/icecc-create-env.in b/client/icecc-create-env.in
index 7255135..ab33ee9 100755
--- a/client/icecc-create-env.in
+++ b/client/icecc-create-env.in
@@ -452,7 +452,7 @@ fi
 
 # special case for weird multilib setups
 for dir in /lib /lib64 /usr/lib /usr/lib64; do
-    test -L $dir && cp -p $dir $tempdir$dir
+    test -L $dir && cp -d $dir $tempdir$dir
 done
 
 new_target_files=
-- 
2.19.1

